node/node.d.ts

```diff
@@ -438,6 +438,7 @@ declare namespace NodeJS {
         unescape: (str: string) => string;
         gc: () => void;
         v8debug?: any;
+        [propName: string]: any;
```

Comment:
@NoHomey I noticed you added stuff to this file . process.exitCode isn't in that file. Could you add it?